desktop/apps/artsy_primer/personalize/stylesheets/categories.styl

```diff
@@ -75,11 +80,17 @@ category-size = 400px
     height 400px
     background-color black
     opacity 0.5
+    @media screen and (max-width: 768px)
+      width 100%
+      height 100vw
   .artsy-primer-personalize-following-icon
     position absolute
     width 400px
     height 400px
     padding-top 45%
+    @media screen and (max-width: 768px)
+      width 100%
+      height 100%
```

Comment:
Just double-checking, is it correct that you’re not using the viewport height here?